src/ts/colorutils.ts

```diff
@@ -7,17 +7,47 @@ export namespace ColorUtils {
     a: number;
```

Comment:
Should we remove all of colorUtils? It's only used by storageUtils but the object we set don't have a `Color` anymore.